src/controllers/wishlist.controller.js

```diff
@@ -137,7 +137,7 @@ const getWishesPerProduct=async(req,res)=>{
     return res.status(200).json(productWishes)
     }
     catch(error){
-        res.status(500).json(error)
+        res.status(500).json({error: error.message.replace(/[^a-zA-Z0-9 ]/g, '')})
```

Comment:
remove this kind of error displaying on response to the user I had designed because it was for Joi error messsage displaying